src/functions-and-arrays.js

```diff
@@ -52,15 +100,39 @@ const wordsUnique = [
   'bring'
 ];
 
-function uniquifyArray() {}
-
-
+function uniquifyArray(array5) {
+  if (array5.length === 0) {
+    return null;
+  }
+  const newUniqueArr = [];
+  for (let i = 0; i < array5.length; i++) {
+    if (array5.indexOf(array5[i]) === array5.lastIndexOf(array5[i])) {
+      newUniqueArr.push(array5[i]);
+    }
+  }
+  return newUniqueArr;
+}
+console.log(uniquifyArray(wordsUnique))
```

Comment:
Hmm we discussed this yesterday. It seems this will copy all the unique elements to `newUniqueArr` and will skip the duplicated ones. I think the requirement for this function is that for duplicate elements, we still need to pick just one of them. So this logic will not do it. 

In this case, I think we could use `indexOf` on the `newUniqueArr` like so:
```
 function uniquifyArray(ar) {
   let newUniqueArr = [];
   for (let el of ar) {
     if (newUniqueArr.indexOf(el) == - 1) {
       newUniqueArr.push(el)
     }
   }
   return newUniqueArr;
 }
```